plugins/storage/pyparts/ceph_daemon_checks.py

```diff
@@ -25,6 +25,44 @@ def __init__(self):
         super().__init__()
         self.ceph_osd_df_tree = self.cli.ceph_osd_df_tree()
         self.ceph_versions = self.cli.ceph_versions()
```

Comment:
Hi Pon, this mad me realise that the existing code needed tidying up and this __init__ method no longer exists. Can you move your ceph_report bits to check_osdmaps_size() and also if you are not actually using the cached ceph_report you don't need to cache it so you can remove that too (from self.cli_cache). Either that or do the json cleanup stuff as part of adding to self.cli_cache and use that instead of calling self.cli directly.